content/iter8 trend/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+date: 2020-08-11T12:00:00+00:00
+menuTitle: iter8-trend
+title: Getting Started with iter8-trend
+weight: 25
```

Comment:
Is this the proper weight? Currently, this means that `iter8 trend` shows up between `Getting started` and `References`.

Reply:
Yes. We've thinking of rearranging things (again). @sriumcp will do most of the changes in his PR. I tried to make the minimal number of changes so as to not force him to refactor a lot.

Reply:
Okay no problem!